functions-and-scope.js

```diff
@@ -73,9 +141,33 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 9
 
 
+//--------------------------------------------------------
+
+// Te nemen stappen:
+// 1- maak een for loop om  alle getallen van de array 1 voor 1 door te lopen
+// 2- leg de waarde van het eerste getal (tijdelijk) vast in een let (variabele grootste getal);
+// 3- check of het volgende getal groter is dan "grootste getal"
+// 3- zo ja, sla dit volgende getal op als "grootste getal".
+// 4- zo nee check dan of het daarna volgende getal groter is dan "grootste getal,
+// 5- nadat de hele loop doorlopen is: return  "grootste getal".
+
+function highestGrade(grades){
+    let highestGrade = grades[0];
```

Comment:
Super dat je met `grades[0]` bent begonnen in plaats van 0. Het is een veelgemaakte fout, dus lekker bezig! 🏅